lib/internal/streams/from.js

```diff
@@ -34,21 +34,49 @@ function from(Readable, iterable, opts) {
     objectMode: true,
     ...opts
   });
+
```

Comment:
An unnecessary change

Reply:
Yep, this can look like unnecessary change when looking over commit. It seems I've added this because I've worked with file as a whole and that's looked better for me.
I'll fix that if you really prefer minimal changes in this PR. Do you?

Reply:
I mean I've added several entities after this line and they are quite big so they are wrapped with empty lines. This line aligns style of variable declaration with those changes.

Reply:
Does it make sense for you?

Reply:
ok, that’s alright